### ISSUE TITLE
Throw takes 1 arg only

### ISSUE THREAD

**Issue by phreed** [shoreleave/shoreleave-remote-ring#13]
I think the parenthesis is misplaced.

https://github.com/shoreleave/shoreleave-remote-ring/blob/master/src/shoreleave/middleware/rpc.clj#L46

```clojure
...
(catch Exception e
        (throw 
             (Exception. (str "Could not locate a namespace when aliasing remotes: " namesp-sym))
             e))
...
```

**Comment by phreed** [comment]
Fixed in pull request #12

**Comment by emanjavacas** [comment]
Given that that pull request hasn't been merged yet, perhaps this issue should be kept open?
